src/v2/view-builder/views/custom-app/ChallengeCustomAppPushView.js

```diff
@@ -0,0 +1,84 @@
+import { _, loc, createButton, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import { BaseForm } from '../../internals';
+import polling from '../shared/polling';
+import { WARNING_TIMEOUT } from '../../utils/Constants';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+
+const warningTemplate = View.extend({
+  className: 'okta-form-infobox-warning infobox infobox-warning',
+  template: hbs`
+    <span class="icon warning-16"></span>
+    <p>{{warning}}</p>
+  `
+});
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    noButtonBar: true,
+
+    className: 'custom-app-push-challenge',
+
+    title() {
+      return loc('oie.verify.custom_app.title', 'login', [this.options.appState.getAuthenticatorDisplayName()]);
+    },
+
+    initialize() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.listenTo(this.model, 'error', this.stopPoll);
+      this.addView();
+    },
+
+    addView() {
+      this.add(createButton({
+        className: 'button button-wide button-primary send-push link-button-disabled',
+        title: loc('oie.custom_app.push.sent', 'login'),
+        click: (e) => {
+          e.preventDefault();
+        }
+      }));
+      this.add(
+        `<span class='accessibility-text' role='alert'>${loc('oie.custom_app.push.sent', 'login')}</span>`,
+      );
+    },
+
+    postRender() {
+      this.startPoll();
+    },
+
+    startPoll() {
+      this.startPolling();
+      this.warningTimeout = setTimeout(_.bind(function() {
+        this.showWarning(loc('oie.custom_app.push.warning', 'login', ));
+      }, this), WARNING_TIMEOUT);
+    },
+
+    stopPoll() {
+      this.stopPolling();
+      this.clearWarning();
+    },
+
+    showWarning(msg) {
+      this.clearWarning();
+      this.add(warningTemplate, '.o-form-error-container', {options: {warning: msg}});
```

Comment:
as part of this ticket IDX is converting all of these harcoded `warning messages` into remediations.. Basically if a view wants to show the warning then it should send it as part of idx response under `messages`.. https://oktainc.atlassian.net/browse/OKTA-408316

can you please ask your backend team to add this warning message into idx response.

Reply:
@kamaldeepcheema-okta is working to make this `ResendView` accepts idx responses instead of hardcoded messages..

Reply:
@pradeepdewda-okta  even if it is purely client side warning ? Here it is just a reminder to check for push notification.

Reply:
@magizh-okta ideally anything that widget renders should be dictated by idx responses.. If widget decides to re-render during poll this message will go away. It works now because on poll widget don't re-render.

Reply:
https://github.com/okta/okta-core/pull/55383

Reply:
interesting... I didn't know this will change to the API response. Good to know!

Reply:
@mauriciocastillosilva-okta  @pradeepdewda-okta  the warning messages remediation might be behind a ff. The rollout is not yet clear.  @pradeepdewda-okta  and @kamaldeepcheema-okta  are working with IDX team to figure it out. I created a follow up ticket for that work https://oktainc.atlassian.net/browse/OKTA-420817

Reply:
@magizh-okta FYI.. @kamaldeepcheema-okta is going to remove FF check and it will not be behind any FFs.

Reply:
Thanks @pradeepdewda-okta . Just to summarize our discussion offline, the work being done for OKTA-412691 will add support to display warning messages based on remediation responses, both at the BaseForm level as well as the special cases in the ResendViews pertaining to OKTA-412691. We will not be gating these changes behind any FF (previously ENG_ENABLE_JWE_STATE_TOKENS_ON_IDX) anymore since these views are OIE specific and the FF will eventually be folded into IDENTITY_ENGINE.

Reply:
@pradeepdewda-okta  I still have some questions on the warning in remediations. https://oktainc.atlassian.net/browse/OKTA-408316?focusedCommentId=2079149

If we are going forward with the remediation approach I have a ticket to address that./
https://oktainc.atlassian.net/browse/OKTA-420817